.gitignore

```diff
@@ -1,8 +1,9 @@
 # Compiled scripts
-Data/Scripts
+
 
 # Packaged artefact
 AutoTest.7z
 
 # Generated README for NexusMods
 README.bbcode
+build.cmd
```

Comment:
I don't understand why do we want to remove this file from git? It is used to generate the compiled scripts, as documented in the readme. I think we should keep it in the source repository.

Reply:
if I (or someone else) modifies the build.cmd to fit their drive structure, then commits those changes then the build.cmd will no longer be correct for your environment, perhaps there is a different way around this, I am far from being a Git expert.  For me it will not make a difference either way as I am now using VSCode and Pyro to compile.

Reply:
Ah ok, I get it. In fact the values in the build scripts are just default values. They can be overridden by setting environment variables of the same name, without changing `build.cmd`.

For example if your game is installed in a different place you can do that:
```
set "gameDir=C:\myOtherPath\Skyrim Special Edition"
build.cmd
```
This will use your own value of `gameDir` in `build.cmd`.

You can also set them among the user or system Windows environment variables from the control panel to have the values stored all the time.